backend/models/index.js

```diff
@@ -5,81 +5,69 @@ const path = require("path");
 const Sequelize = require("sequelize");
 const basename = path.basename(__filename);
 const dbConfig = require(__dirname + "/../config/database");
-const db = {};
 
-const sequelizeTelemetryBackend = new Sequelize(
-  dbConfig.telemetryDatabase.database,
-  dbConfig.telemetryDatabase.username,
-  dbConfig.telemetryDatabase.password,
-  {
-    ...dbConfig.telemetryDatabase,
-    logging: false,
-  }
-);
+let _exports = {};
+
+const sequelizeTelemetryBackend = dbConfig.writeOnlyTelemetryDatabase.host
```

Comment:
Why the variable below called `sequelizeTelemetryBackendReadOnly`, but this one is not `sequelizeTelemetryBackendWriteOnly`?

Reply:
I did not want to refactor the whole world. My main concern was that we actually used the same telemetry instance for writes and reads due to re-using the same configuration options, and I actually wanted to fix that.